app/views/moderations/actions_panel.html.erb

```diff
@@ -101,7 +101,7 @@
       </button>
 
       <div id="adjust-tags-options" class="adjust-tags-options dropdown-options hidden">
-        <% if current_user.any_admin? %>
+        <% if policy(@moderatable).adjust_all_tags? %>
```

Comment:
This check specifically asks if the user can moderate ALL tags or JUST the tags that they are authorized to moderate (tag moderators).